Sources/Sentry/SentryTimeToDisplayTracker.m

```diff
@@ -53,6 +53,13 @@ - (instancetype)initForController:(UIViewController *)controller
     return self;
 }
 
+- (instancetype)initWithName:(NSString *)name waitForFullDisplay:(BOOL)waitForFullDisplay
+{
+    return [self initWithName:name
+           waitForFullDisplay:waitForFullDisplay
+         dispatchQueueWrapper:SentryDependencyContainer.sharedInstance.dispatchQueueWrapper];
```

Comment:
`m`: Please stick to the pattern of passing in the `dispatchQueueWrapper` as a dependency.

Reply:
This is a convenient init so we dont have to re-define SentryDispatchQueue for SentrySwiftUI, which is a different module, it does not have access to everything inside of Sentry.